docs/gettingstarted.md

```diff
@@ -0,0 +1,22 @@
+# gulp-rev
+## Install
+
+```
+$ npm install --save-dev gulp-rev
+```
+
+
+## Usage
+
+```js
+var gulp = require('gulp');
+var rev = require('gulp-rev');
+
+gulp.task('default', function () {
+	return gulp.src('src/*.css')
+		.pipe(rev())
+		.pipe(gulp.dest('dist'));
+});
+```
```

Comment:
Yes, but list the different documentation documents in the `Docs` section in the readme directly.